### PR TITLE
Virtual Classroom - Backend

### PR DESCRIPTION
**Pull request by PravinKumarRajaSahayam**
I created a new branch master and added some controller source file into that Git Repository.
Could you review my Controller source file?

### CODE REVIEW

File: LoginController.java [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+@RestController
+public class LoginController 
+{
+	@Autowired
+	private UserService userService;
+	
+	@Autowired
+	private TeacherService teacherService;
+	
+	//welcome page
```
Comment (Gayathrik05) [inline LoginController.java:23]: Give a mapping code for welcome page

**Review by Gayathrik05 — COMMENTED** [review]
I reviewed your check your login controller.java file . Please check it and re-code it.

File: LoginController.java (hunk 1 above)
Comment (PravinKumarRajaSahayam) [inline LoginController.java:23]: @GetMapping("/")
    public String welcomeMessage()
    {
    	return "Welcome to Virtual Classroom !!!";
    }

Comment (PravinKumarRajaSahayam) [reply LoginController.java:23]: I edited my Login Controller source file for Mapping the welcome page.
Could you recheck review my code

Comment (PravinKumarRajaSahayam) [reply LoginController.java:23]: I edited my Login Controller source file for Mapping the welcome page.
Could you recheck review my code

Comment (Gayathrik05) [reply LoginController.java:23]: I reviewed your code. Please add this code into your file.

**Review by PravinKumarRajaSahayam — COMMENTED** [review]
I edited my Login Controller source file for Mapping the welcome page.
Could you recheck review my code

File: LoginController.java (hunk 1 above)
Comment (Gayathrik05) [reply LoginController.java:23]: Ok. You can update your file.

**Review by Gayathrik05 — COMMENTED** [review]
Ok. You can update your file.

File: RegistrationController.java [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+@RestController
+public class RegistrationController 
+{
+	@Autowired
+	private UserService userService;
+	
+	@Autowired
+	private TeacherService teacherService;
+	
+	//register user
```
Comment (Gayathrik05) [inline RegistrationController.java:22]: Please add mapping file for register user.

**Review by Gayathrik05 — COMMENTED** [review]
In Register Controller.java file, add a mapping file for that register user.

File: LoginController.java (hunk 1 above)
Comment (PravinKumarRajaSahayam) [reply LoginController.java:23]: Ok sure

**Review by PravinKumarRajaSahayam — COMMENTED** [review]
Ok sure

File: RegistrationController.java (hunk 2 above)
Comment (PravinKumarRajaSahayam) [reply RegistrationController.java:22]: I added the Post Mapping for register user and upload the source file into my Repository. Could you recheck my source file

**Review by PravinKumarRajaSahayam — COMMENTED** [review]
I added the Post Mapping for register user and upload the source file into my Repository. Could you recheck my source file

File: RegistrationController.java (hunk 2 above)
Comment (Gayathrik05) [reply RegistrationController.java:22]: Ok you can update.

**Review by Gayathrik05 — DISMISSED** [review]

File: RegistrationController.java (hunk 2 above)
Comment (PravinKumarRajaSahayam) [reply RegistrationController.java:22]: Ok sure

File: TeacherController.java [hunk 3]
```diff
@@ -0,0 +1,216 @@
...
+	@Autowired
+	private CourseService courseService;
+	
+	@Autowired
+	private ChapterService chapterService;
+	
+	@Autowired
+	private WishlistService wishlistService;
+	
+	//teacher list
```
Comment (Gayathrik05) [inline TeacherController.java:41]: list all the teachers details.

**Review by PravinKumarRajaSahayam — COMMENTED** [review]
Ok sure, I will update my source code

**Review by Gayathrik05 — COMMENTED** [review]
list all the teachers details.

**Review by PravinKumarRajaSahayam — COMMENTED** [review]
I added the Get Mapping for list out all the teacher data.
Could you recheck my sorce code

**Review by Gayathrik05 — COMMENTED** [review]
Okay. i approve you files to be add in to your main repository

**Pull request merged**